google-cloud-firestore/src/main/java/com/google/cloud/firestore/UpdateBuilder.java

```diff
@@ -43,17 +52,62 @@
  * Abstract class that collects and bundles all write operations for {@link Transaction} and {@link
  * WriteBatch}.
  */
-public abstract class UpdateBuilder<T extends UpdateBuilder> {
+public abstract class UpdateBuilder<T> {
```

Comment:
Please add the `@InternalExtensionOnly` (from `com.google.api.core`) annotation to this class so we're covered should we need to modify anything

Reply:
Done.